crates/holochain_conductor_api/src/config/conductor/dpki_config.rs

```diff
@@ -41,27 +38,25 @@ pub struct DpkiConfig {
     pub allow_throwaway_random_dpki_agent_key: bool,
 
     /// For testing only, we can turn off DPKI if needed.
-    /// TODO: this can be removed once DPKI is truly optional again.
+    // TODO: this can be removed once DPKI is truly optional again.
     #[serde(default)]
     pub no_dpki: bool,
 }
 
 impl DpkiConfig {
-    pub fn production(dna_path: Option<PathBuf>) -> Self {
-        Self {
-            dna_path,
-            network_seed: DPKI_NETWORK_SEED_MAIN.to_string(),
-            allow_throwaway_random_dpki_agent_key: false,
-            no_dpki: false,
-        }
-    }
-
```

Comment:
I suggest to remove this impression of a production DPKI network.

Reply:
Why? This is indeed what the conductor should be using for real-world apps, when DPKI is ready to go.

Reply:
Yes, when DPKI is ready to go, but as DPKI is not ready to go, I suggest to remove it. If everyone else wants to keep it in, I'm happy to keep it in.

Reply:
Yeah I'd prefer that when people turn this unstable feature back on, they get what they'd get before this PR. I don't think, unless something is really wrong, that we should get into changing the code